db/migrations/20170821165118_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.raw('UPDATE movies SET name = title');
```

Comment:
Depending on the size of the table, we'll batch this so that a long migration doesn't block incoming requests. There are plenty of examples of this in API so definitely not necessary here.